tachyon/crypto/commitments/pedersen/pedersen.h

```diff
@@ -92,7 +92,7 @@ class Pedersen final
       return;
     }
 #endif
-    cpu_batch_commitments_.resize(size);
+    cpu_batch_commitments_.resize(size, Bucket::Zero());
```

Comment:
It seems like we don't need to initialize the values, since `cpu_batch_commitments_` will be filled with the values. But just add a comment that why this is safe.